src/sparqlClient/Statement.php

```diff
@@ -67,9 +67,8 @@ public function fetch(int $fetchStyle = PDO::FETCH_OBJ): object | array | false
                     return $row[array_keys($row)[0]];
                 default;
             }
-        } else {
-            return false;
         }
+        return false;
```

Comment:
IMHO there should be rather an exception "unsupported fetch style" thrown as the `case` statement default option.

Reply:
An exception is also fine.

Reply:
Than rejecting this pull request. The solution with exception is implemented by 00bdb846da386238dd38279182403b12aa684a16